www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -100,11 +100,12 @@ const MultilabelButtonGroup = ({ trip, buttonsInline=false }) => {
           )
         })}
       </View>
-      <View>
-        <IconButton icon='check-bold' mode='outlined' size={16} onPress={verifyTrip}
-                    disabled={trip.verifiability != 'can-verify'}
-                    style={{width: 20, height: 20, margin: 3}}/>
-      </View>
+      {trip.verifiability === 'can-verify' && (
+        <View style={{marginTop:'1rem'}}>
+          <IconButton icon='check-bold' mode='outlined' size={18} onPress={verifyTrip}
+                       style={{width: 24, height: 24, margin: 3, backgroundColor: colors.secondaryContainer}}/>
```

Comment:
```suggestion
          <IconButton icon='check-bold' mode='outlined' size={18} onPress={verifyTrip}
                      containerColor={colors.secondaryContainer}
                      style={{width: 24, height: 24, margin: 3, borderColor: colors.secondary}} />
```
I made 2 suggestions here:

1. There actually exists a dedicated property on `IconButton` called `containerColor` that we can use instead of `style.backgroundColor`
https://callstack.github.io/react-native-paper/docs/components/IconButton/#containercolor Although setting `backgroundColor` does the exact same thing in this instance, it's probably best practice to use the property since it's there. React Native Paper is well-planned component library and there might be a good reason that the developers included a property for this.
1. I recommend coloring the border of the checkmark button to match the border of the yellow labels.